### ISSUE TITLE
Strong naming

### ISSUE THREAD

**Issue by repko-artem** [pootzko/InfluxData.Net#76]
Thanks lot for the library!
Do you have any plans to sign your library? It's not possible now to use your NuGet package in solutions that require strong naming.

**Comment by tihomir-kit** [comment]
Yeah, it has been on my mind for some time, I should do it. I'll do it for the next release.

**Comment by GeorgeHahn** [comment]
If you do this, please retain a non-strong named version on Nuget. It looks like many projects append [.StrongName](https://www.nuget.org/packages?q=.strongname) to their strong named binaries. This is desirable for a number of reasons that I won't go into here - but there's lots of discussion around the web:

https://github.com/reactiveui/ReactiveUI/issues/43
https://github.com/libgit2/libgit2sharp/issues/212
https://github.com/ericsink/SQLitePCL.raw/issues/53
https://github.com/octokit/octokit.net/issues/405
https://github.com/twilio/twilio-csharp/issues/298#issuecomment-285473282

**Comment by tihomir-kit** [comment]
Great, info, didn't know that. I'll check it out, thanks.

**Comment by mdy6** [comment]
Hi,
I want to use your library, but i have the same Strong naming issue.
Have you already figure out this  ?
Thanks

**Comment by GillesTourreau** [comment]
Hi @pootzko,

We have the same issue here...
All our projects are signed with strong name and we can not use directly the library...
We need to get all the source code of InfuxData.Net project and compile with a strong name in our side... :-(

Hope that you will be able to fix this issue soon...

**Comment by dprothero** [comment]
There are various options you can take to deal with this issue:

https://support.twilio.com/hc/en-us/articles/360008375953-Does-Twilio-provide-a-strong-named-assembly-of-the-C-helper-library-
